src/libraries/Common/tests/System/Security/Cryptography/MLKemBaseTests.cs

```diff
@@ -48,8 +48,8 @@ public void ExportPrivateSeed_OnlyHasDecapsulationKey()
         {
             using MLKem kem = ImportDecapsulationKey(MLKemAlgorithm.MLKem512, MLKemTestData.MLKem512DecapsulationKey);
 
-            Assert.Throws<CryptographicException>(() => kem.ExportPrivateSeed());
-            Assert.Throws<CryptographicException>(() => kem.ExportPrivateSeed(
+            Assert.ThrowsAny<CryptographicException>(() => kem.ExportPrivateSeed());
```

Comment:
_Kevin, why did these change to `ThrowAny`?_

Because our throw helper throws two different exceptions, depending on if you are .NET Framework or .NET:

https://github.com/dotnet/runtime/blob/5415b7342d44af9c974905760539f198fad13682/src/libraries/Common/src/System/Security/Cryptography/CryptoThrowHelper.Windows.cs#L28-L40

This is going to improve when https://github.com/dotnet/runtime/issues/116509 is implemented.